packages/kbn-type-summarizer/src/lib/type_summary/type_summary_namer.ts

```diff
@@ -0,0 +1,67 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License
+ * 2.0 and the Server Side Public License, v 1; you may not use this file except
+ * in compliance with, at your election, the Elastic License 2.0 or the Server
+ * Side Public License, v 1.
+ */
+
+import ts from 'typescript';
+
+import { AstIndex } from '../ast_index';
+
+const INVALID_NAMES = ['default', 'import'];
```

Comment:
is there other reserved words that could be INVALID names like `export`?

Reply:
Yeah, didn't spend time worrying about finding all of them but I'll add `export` to the list.